lib/srv/sess.go

```diff
@@ -1380,7 +1392,7 @@ func (s *session) startInteractive(ctx context.Context, scx *ServerContext, p *p
 		go func() {
 			// Close the BPF recording session once the session is closed
 			<-s.stopC
-			err = scx.srv.GetBPF().CloseSession(sessionContext)
+			err = bpfService.CloseSession(sessionContext)
```

Comment:
nit: would be better nested in the following if block with `:=` to respect the smallest scope principle.